src/test/java/org/jenkinsci/plugins/badge/WorkflowBadgeActionTest.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * The MIT License
+ *
+ * Copyright 2013 Dominik Bartholdi.
```

Comment:
Actually, on second thought.... I think it is okay to remove that since it's not on *all* files and this is new code.